dev-libs/soci/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>WolfgangESanyer@Gmail.com</email>
```

Comment:
This isn't your git author email btw

Reply:
Hrm, that's right, I had to locally change my git author email to `ezzieyguywuf@gmail.com` to match my bugzilla thing.

I guess I should use the same email address here for consistency?

Reply:
Understood, I'll update this to my bugzilla email.

Reply:
yes, I don't really mind which, but they should be consistent

Reply:
Fixed, changed it to the bugzilla email.